CONTRIBUTING.md

```diff
@@ -4,42 +4,22 @@ When it comes to open source, there are different ways you can contribute, all
 of which are valuable. Here's a few guidelines that should help you as you
 prepare your contribution.
 
-- [Setup](#setup)
-- [Development](#development)
-  - [Tooling](#tooling)
-  - [Commands](#commands)
-    - [Package Aliasing and Yarn Workspace](#package-aliasing-and-yarn-workspace)
-  - [Documentation](#documentation)
-  - [Storybook](#storybook)
-- [Think you found a bug?](#think-you-found-a-bug)
-- [Proposing new or changed API?](#proposing-new-or-changed-api)
-- [Making a Pull Request?](#making-a-pull-request)
-  - [Commit Convention](#commit-convention)
-  - [Steps to PR](#steps-to-pr)
-  - [Tests](#tests)
-- [Want to write a blog post or tutorial](#want-to-write-a-blog-post-or-tutorial)
-- [Want to help improve the docs?](#want-to-help-improve-the-docs)
-- [License](#license)
```

Comment:
Do we really want to get rid of the ToC? Seems like it's potentially helpful

Reply:
I feel it gets in the way of the most important information needed to contribute.

I've tried to onboard 2 friends and it initially felt like a lot of information just to contribute. I thought to limit that to just a few steps.

It's basically a test for "less is more". I can always regenerate it if we have a need for it.

Reply:
Yeah I fully agree with the "less is more" approach!